sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/Azure.AI.DocumentTranslation.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" VersionOverride="12.8.0" />
```

Comment:
Our repo already uses this version. See [here](https://github.com/Azure/azure-sdk-for-net/pull/19778).
So you could do `<PackageReference Include="Azure.Storage.Blobs" />`

Reply:
@maririos when I remove this it uses 12.6.0

Reply:
my link was wrong. I meant https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L85

Reply:
oh wait I see https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L154

Reply:
let me follow up

Reply:
It looks like we can update it only if the build is not broken. So could you give it a try?  the update will be in here  https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L154